client/src/components/buttons/Button.js

```diff
@@ -9,8 +9,10 @@ const Multiplebuttonsdiv = styled.div`
   display: flex;
   flex-direction: row;
   cursor: pointer;
-  justify-content: space-around;
-  width: 100%;
+  margin-left: 5%;
+  width: 90%;
+  max-width: 90%;
+  overflow-x: scroll;
```

Comment:
Personally I'm not a fan of having a scroll. From a UX perspective I think being able to view all the buttons at once and then selecting the one you want is more effective than scrolling to find read then scrolling back to select.